CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 *                  @MikaelSmith
 */wash.*           @puppetlabs/wash
 */puppet-bolt.*    @puppetlabs/bolt
-*/pdk.*            @puppetlabs/pdk
+*/pdk.*            @puppetlabs/modules
```

Comment:
git doesn't seem sure this team exists? Or we might not have write access to the repo

Reply:
yeah the team exists. We need write access to the repo then which i think only maintainers of this repo can give